api/v1alpha1/dnspolicy_types.go

```diff
@@ -134,19 +139,14 @@ type LoadBalancingGeo struct {
 
 // DNSPolicyStatus defines the observed state of DNSPolicy
 type DNSPolicyStatus struct {
+	reconcilers.StatusMeta `json:",inline"`
```

Comment:
Should `StatusMeta` not be in the versioned API? What is the purpose of it? Is it intended to be the common expected fields for the status on all policies? If so, should "Conditions" be in there also?

Reply:
you are right, this is leftovers of a previous try that did not succeed. I will remove it.